src/com/lessons/hw/Employee.java

```diff
@@ -0,0 +1,48 @@
+package com.lessons.hw;
+
+public class Employee {
+    private String full_name;
```

Comment:
нужно использовать camelCase стиль для именования в java
```suggestion
    private String fullName;
```